src/views/RoomView.vue

```diff
@@ -161,11 +191,90 @@
 
   channel.loadWebcams();
 
+  function updateWhiteboardSize() {
+    width.value = document.getElementById('whiteboard-wrapper').clientWidth;
+    height.value = document.getElementById('whiteboard-wrapper').clientHeight;
+  }
+
   function toggleVideo() {
     channel.toggleVideo();
   }
 
   function toggleAudio() {
     channel.toggleAudio();
   }
+
+  function toggleWhiteboard() {
+    if (showNotes.value) toggleNotes();
+    showWhiteboard.value = !showWhiteboard.value;
+    setTimeout(() => {
+      updateWhiteboardSize();
+    }, 50);
+  }
+
+  function toggleExpandWhiteboard() {
+    expandWhiteboard.value = !expandWhiteboard.value;
+    setTimeout(() => {
+      updateWhiteboardSize();
+    }, 50);
```

Comment:
wieso hier die timeouts?

Reply:
Ja ohne hat das nicht korrekt funktioniert, in der updateWhiteboardSize wird ja die Größe der dom-elemente genutzt und dass die korrekt war, waren die Timeouts notwendig. Ich guck nochmal drüber ob's da noch ne bessere Lösung gibt.